app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_list = command.split(" ")
```

Comment:
Space is the default value for split() don't write it, and it's better to unpack list like this:

```suggestion
    command, existing_file, new_file = command.split()
```